zerver/models.py

```diff
@@ -1248,6 +1251,7 @@ class PreregistrationUser(models.Model):
     # store it here to use it to prepopulate the Full Name field in the registration form:
     full_name = models.CharField(max_length=UserProfile.MAX_NAME_LENGTH, null=True)  # type: Optional[str]
     full_name_validated = models.BooleanField(default=False)
+    user_avatar_url = models.CharField(max_length=UserProfile.MAX_URL_LENGTH, null=True)
```

Comment:
The url doesn't get used in the UserProfile (because we upload the image), so the MAX_URL_LENGTH constant should probably be in the PreregUSer class too.